setup.py

```diff
@@ -70,6 +70,7 @@
     "sphinx-rtd-theme",
     "wheel>=0.36.2",
     "matplotlib>=3.0.0",
+    "pydantic==1.9.1",
```

Comment:
If possible I'd advise something a smidge looser than this. Otherwise the other repos we have will also be forced to use pydantic 1.9.1 ... which at the moment is fine, but maybe it's better to introduce it as a requirement in a loose way? I think I'm mostly thinking something like
```suggestion
    "pydantic~=1.9.1",
```